# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1358222214 18000
#      Mon Jan 14 22:56:54 2013 -0500
# Branch classdef
# Node ID 9c7e1c633e477807ef900b3ca2365cca8ab180f1
# Parent  dd4258398a59d0a63c545d67c6594922ddb6ac68
Include classdef tests in Makefile.am

* test/Makefile.am: Include classdef tests.
* test/classdef/test_classdef.m: Renamed test/classdef/test_classes.m.

diff --git a/test/Makefile.am b/test/Makefile.am
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -47,16 +47,17 @@ FCN_FILES = \
   test_system.m \
   test_transpose.m \
   test_try.m \
   test_unwind.m \
   test_while.m
 
 include bug-35448/module.mk
 include bug-36025/module.mk
+include classdef/module.mk
 include classes/module.mk
 include class-concat/module.mk
 include ctor-vs-method/module.mk
 include fcn-handle-derived-resolution/module.mk
 include nest/module.mk
 
 check: test_sparse.m test_bc_overloads.m
 	$(top_builddir)/run-octave --norc --silent --no-history $(srcdir)/fntests.m $(srcdir)
diff --git a/test/classdef/module.mk b/test/classdef/module.mk
--- a/test/classdef/module.mk
+++ b/test/classdef/module.mk
@@ -1,5 +1,5 @@
-classes_FCN_FILES = \
+classdef_FCN_FILES = \
   classes/foo_payment.m \
-  classes/test_classes.m
+  classes/test_classdef.m
 
-FCN_FILES += $(classes_FCN_FILES)
+FCN_FILES += $(classdef_FCN_FILES)
diff --git a/test/classdef/test_classes.m b/test/classdef/test_classdef.m
rename from test/classdef/test_classes.m
rename to test/classdef/test_classdef.m
